nautobot/docs/user-guide/core-data-model/dcim/hardwarefamily.md

```diff
@@ -0,0 +1,3 @@
+# Hardware Family
```

Comment:
Probably would be good to add some verbiage to `devicetype.md` referencing this model. Also should probably have a `+++ 2.2.0` annotation in this file and that one.